src/data_info/services/path_lists.clj

```diff
@@ -106,7 +106,15 @@
       (throw+ {:error_code ERR_NOT_FOUND
                :reason "No paths matched the request."}))
 
-    (string/join "\n" (concat [(cfg/path-list-file-identifier)] filtered-paths))))
+    (string/join "\n" (concat [path-list-file-identifier] filtered-paths))))
+
+(defn- path-list-type->info-type+file-identifier
+  "Returns a list containing the appropriate Path List info-type and file identifier header string
+   for the given `path-list-type` request parameter."
+  [path-list-type]
+  (if (= path-list-type :ht)
```

Comment:
Can this be changed to use actual `infotype` rather than `ht` just to be consistent with what we are checking for ?

Reply:
Good idea 💡 
I'll change it.